pkg/virt-operator/creation/components/config_maps.go

```diff
@@ -0,0 +1,60 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2020 Red Hat, Inc.
+ *
+ */
+package components
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+//NewKubevirtConfigMap returns base config map
+func NewKubevirtConfigMap(namespace string) *corev1.ConfigMap {
+	return &corev1.ConfigMap{
+		TypeMeta: metav1.TypeMeta{
+			Kind:       "ConfigMap",
+			APIVersion: "v1",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Namespace: namespace,
+		},
+	}
+
+}
+
+//NewNodeLabellerConfigMap returns configmap with configuration of obsolete cpus and minCPU for node labeller
+func NewNodeLabellerConfigMap(namespace string) *corev1.ConfigMap {
+	cm := NewKubevirtConfigMap(namespace)
+	cm.Name = "kubevirt-cpu-plugin-configmap"
+	cm.Data = map[string]string{
+		"cpu-plugin-configmap.yaml": `obsoleteCPUs:
```

Comment:
maybe this is a dumb question.

Do we need this configmap? or can these values all be built into virt-handler statically? since the operator is creating this configmap, it's not something people can modify directly. It's owned by the operator. So if people aren't modifying this configmap, and only virt-handler needs to know this information, what's the purpose of the configmap?

Reply:
User can modify this config map and sets which cpu models will be black listed, or which cpu features are taken as "basic" cpu features

Reply:
umm. Well something to think about here. When someone updates KubeVirt, that configmap is going to get reset (operator will stomp on the values) which will erase any user set values.

Any ideas on how to support user settings in a way where we won't erase later on?

Reply:
Does kubevirt has any functionality which sets: Do not touch on this configmap during update?

Reply:
Can we merge this config map with our kubevirt configmap?

Reply:
We can, but issue might be, that it will be incompatible with previous versions of labeller. But AFAIK this should not be so big problem, because ssp operator didn't allow to update this config map, due to reconcile loop. So it keeps reseting config map to its original state.

Reply:
Was it possible to configure the operator to set it to specific values? If not, we can probably merge them. Will you need to express the default configuration explicitly, or can it be implicitly and moved over time with kubevirt releases?

Reply:
It wasn't possible to set it via running operator. Labeller config values can stay in kubevirt config